### ISSUE TITLE
Add button

### ISSUE THREAD

**Issue by HillNathan** [HillNathan/columbarium-project#14]
Add a button that says "Reserve this Plot" for people who want to reserve a plot. 

- Probably want to make a "confirm action" for this before going ahead and doing the following:
   > Update the database with an "on hold" or "pending" (currently shows pending, will eventually update to show on hold)
   > link opens in new window to JotForm to fill out reservation information so that the office is notified of the interest in the columbarium. Nat would like to have to Plot# field pre-filled, which is possible by using url parameters. 
          + https://www.jotform.com/help/71-Prepopulating-Fields-to-Your-JotForm-via-URL-Parameters

**Comment by HillNathan** [comment]
Button is added, as well as the routine to send the info to the DB  to mark the plot as on hold and to open up the jotform website so they can notify the office of their intent to hold the plot. 

I am still waiting on the jotform to be done so it can be linked, and I also need to add a "Please confirm message" box with more information about what the next steps are for the process, so leaving this issue here for now.
